### ISSUE TITLE
Fluent Api Configuration for SqlServer

### ISSUE THREAD

**Issue by srzsantiago** [thepirat000/Audit.NET#670]
**Describe the bug**
After upgrading from **Audit.Net.SqlServer** version 24.0.1 to 25.0.x, the fluent API configuration began failing. The issue specifically occurs with the dynamic schema configuration, where the **AuditEventWebApi** consistently results in being null. I've attempted alternative approaches, such as using **SqlDataProvider**, but the problem persists—the event is always null. Despite searching for breaking changes between versions 24.x and 25.x, I haven't found any relevant information. It's essential for me to use a dynamic schema since the database contains multiple schemas. Below is my configuration with the fluent API:

```
Audit.Core.Configuration.Setup()
    .UseSqlServer(config => config
        .ConnectionString(c => GetAuditConnectionString((AuditEventWebApi)c))
        .Schema(s => GetAuditTableSchema((AuditEventWebApi)s))
        .TableName(t => GetAuditTableName((AuditEventWebApi)t))
        .IdColumnName("IdColumn")
        .JsonColumnName("JsonColumn")
    );
```

In this configuration, the variable **s** within the schema method is always null.

**To Reproduce**
Trigger a Web API Action.

**Expected Behavior**
The **AuditEventWebApi** variable **s** should be populated with an AuditApiAction containing all the information about the Web API action.

**Libraries**
**Audit.NET**: 25.0.4
**Audit.NET.SqlServer**: 25.0.4
**Audit.WebApi.Core**: 25.0.4

**Target .NET Framework**
.NET 8.0

**Comment by thepirat000** [comment]
It's a bug in the condition on the following line:

https://github.com/thepirat000/Audit.NET/blob/e505b7797bec523042d1fd4f1d12e7e40e915088/src/Audit.NET.SqlServer/Providers/SqlDataProvider.cs#L174-L176

It should be: 

```c#
            return Schema.GetValue(auditEvent) != null  
                ? ...
                : ...;
```

I will be fixing this soon and creating a new version.

**Comment by thepirat000** [comment]
Fixed on version 25.0.5, please upgrade your references and re-test

**Comment by srzsantiago** [comment]
I upgraded and re-tested it. It all works! Thank you very much for the fast reaction and implementation.
